templates/bin/check

```diff
@@ -1,28 +1,12 @@
 #!/bin/sh
 
-if ! grep --exclude-dir="app/assets/typings/**" -i -r 'console.log' app spec
-then
-  echo 'console.log found. Please fix them and try again, commit aborted'
-  exit 1
-fi
+set -e
 
-if ! bundle exec mdl ./
-then
-  echo 'Violated markdown rules, see https://github.com/mivok/markdownlint/blob/master/docs/RULES.md, commit aborted'
-  exit 1
-fi
+bin/fastcheck
 
-if ! bundle exec rubocop -D -c .rubocop.yml --fail-fast
-then
-  echo 'rubocop detected issues!'
-  bundle exec rubocop -a -D -c .rubocop.yml
-  echo 'Tried to auto correct the issues, but must be reviewed manually, commit aborted'
-  exit 1
-fi
-
-if ! bundle exec slim-lint app/views/ -c .slim-lint.yml
+if ! grep --exclude-dir="app/assets/typings/**" -i -r 'console.log' app spec
 then
-  echo 'slim-lint detected issues, commit aborted'
+  echo 'console.log found. Please fix them and try again, commit aborted'
```

Comment:
why did you keep this?

Reply:
Because the grep command does not output why it failed. Therefore, you would have no idea why the CI broke